<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>411</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="AbeforeGridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="AXbeforeLineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="AYbeforeLineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>531</x>
      <y>350</y>
      <width>110</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>490</y>
      <width>411</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="CgridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="CXLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="CYLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>411</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLineEdit" name="BXbeforeLineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="BYbeforeLineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>14</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="operatorGridLayout">
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_10">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>+</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_7">
       <property name="text">
        <string>*=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_8">
       <property name="text">
        <string>/=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>+=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_6">
       <property name="text">
        <string>-=</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_9">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_11">
       <property name="text">
        <string>&lt;=</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="radioButton_12">
       <property name="text">
        <string>&gt;=</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>280</y>
      <width>410</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLineEdit" name="BXafterLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="BYafterLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>410</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="AafterGridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="AXafterLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="AYafterLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>210</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>915</x>
      <y>210</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="AincrRadioButton_1">
        <property name="text">
         <string>A++</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="AincrRadioButton_3">
        <property name="text">
         <string>++A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="AincrRadioButton_2">
        <property name="text">
         <string>A--</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="AincrRadioButton_4">
        <property name="text">
         <string>--A</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="AincrRadioButton_0">
        <property name="text">
         <string>None</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Incrementation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="1" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="BincrRadioButton_4">
        <property name="text">
         <string>--B</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="BincrRadioButton_1">
        <property name="text">
         <string>B++</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="BincrRadioButton_2">
        <property name="text">
         <string>B--</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="BincrRadioButton_3">
        <property name="text">
         <string>++B</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="BincrRadioButton_0">
        <property name="text">
         <string>None</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Incrementation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>450</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
